kolibri/plugins/coach/assets/src/views/common.js

```diff
@@ -219,6 +220,10 @@ export default {
     },
   },
   methods: {
+    // This is a safer way to get the content kind to quickly patch #6552
+    contentIdIsForExercise(contentId) {
+      return get(this.contentMap, [contentId, 'kind']) === 'exercise';
+    },
```

Comment:
This works, but seems highly specific. Maybe a more generally useful function would be 

```javascript
    getContentKind(contentId) {
      return get(this.contentMap, [contentId, 'kind']);
    },
```

This will return `undefined` when the ID doesn't exist, and then below you can write e.g.

```javascript
this.getContentKind(status.content_id) === 'exercise' && //...
```

Reply:
Yeah, it's pretty specific because the coach reports make a pretty fundamental divide between exercises and non-exercises.